Foundation/NSDictionary.swift

```diff
@@ -33,7 +33,12 @@ open class NSDictionary : NSObject, NSCopying, NSMutableCopying, NSSecureCoding,
     }
     
     open func value(forKey key: String) -> Any? {
-        NSUnsupported()
+        if key.hasPrefix("@") {
```

Comment:
It is part of our API, but I'm kinda conflicted bringing this in; as it is, we do not have the ability to implement the full contract of KVC in Swift, and I don't know if the presence of a method is dangerous enough.

I would probably at the very least have some sort of diagnostic (via deprecation maybe?) on usage of this method. But that's for another patch.